workflows/downstream/downstream.wdl

```diff
@@ -146,6 +147,24 @@ workflow downstream {
       runtime_attributes    = default_runtime_attributes
   }
 
+  Array[File] cpg_pileup_beds = select_all([
+    cpg_pileup.combined_bed,
+    cpg_pileup.hap1_bed,
+    cpg_pileup.hap2_bed
+  ])
+
+  if (length(cpg_pileup_beds) > 0) {
```

Comment:
question: is this check basically for datasets without methylation tags?

Reply:
Right, or with coverage so low that pileup beds are not created.